include/bluetooth/addr.h

```diff
@@ -40,44 +41,89 @@ typedef struct {
 	bt_addr_t a;
 } bt_addr_le_t;
 
+/* Bluetooth device "any" address, not a valid address */
```

Comment:
Do we want these descriptions to be caught by Doxygen? Shoulnd't they start with `/**` then (Similar to the additional documentation you added in `uuid.h`).